### PR TITLE
Completed css-positioning exercise.

### PR DESCRIPTION
**Pull request by saraysc**
image
image
image

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  bottom: 3px;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+  width: 300px;
+  height: 300px;
+
```
Comment (Zhiwutian) [inline css-positioning/absolute.css:42]: this element should be position absolute

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update

File: css-positioning/absolute.css (hunk 1 above)
Comment (saraysc) [reply css-positioning/absolute.css:43]: image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**